src/content/index.md

```diff
@@ -108,58 +108,58 @@ But the snail is much faster.
 
 ![page 13](../images/page13.png)
 
-“Don’t finish it: we can still be friends!”  
+“Don’t finish it! We can still be friends!”  
 Tries Hector one more time  
-Desperately...
+desperately...
 
 ![page 14](../images/page14.png)
 
 ---
 
-Once again, Hector was too slow  
-There’s nothing left  
-The salad is gone  
-And there’s no more food around him.
+Once again, Hector was too slow.  
+There’s nothing left.
+The salad is gone
+and there’s no more food around him.
 ![page 15](../images/page15.png)
 
 ---
 
-When a little mouse  
-Comes to offer him a nut.  
-“I’m small and fast, I can give you more.  
+Then a little mouse  
+comes to offer him a nut.  
+“I’m small and fast, and I can get you more nuts.  
```

Comment:
Ha! Is it indispensable to add 'nuts' after 'more'? I liked that it rhymed with 'protector'!